src/main/java/com/fontys_automotive/api/tus/models/project/Function.java

```diff
@@ -1,15 +1,19 @@
 package com.fontys_automotive.api.tus.models.project;
 
-import java.util.ArrayList;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
 
+import java.util.ArrayList;
+@Data
 public class Function{
     public String title;
-    public String level_modifier;
+    @JsonProperty("level_modifier")
+    public String levelModifier;
     public ArrayList<Classification> classifications;
 
-    public Function(String title, String level_modifier, ArrayList<Classification> classifications) {
+    public Function(String title, String levelModifier, ArrayList<Classification> classifications) {
```

Comment:
Deze constructor is overbodig aangezien hier gebruik gemaakt wordt van de `@Data` annotation

Reply:
Indien er geen allArgsConstructor annotation aanwezig is, is een normale constructor vereist